python_modules/dagster/setup.py

```diff
@@ -68,8 +68,8 @@ def get_version():
             # core (not explicitly expressed atm)
             "alembic>=1.2.1",
             "croniter>=0.3.34",
-            "grpcio==1.30.0",  # pinned so that code generated by grpc will work
-            "protobuf==3.12.4",  # pinned so that code generated by grpc will work
+            "grpcio>=1.32.0",  # ensure version we require is >= that with which we generated the grpc code (set in dev-requirements)
```

Comment:
Is it asking too much to have a test which checks this against dev-requirements?